test/publishing_api_v2_test.rb

```diff
@@ -511,6 +511,152 @@ def content_item_for_content_id(content_id, attrs = {})
     end
   end
 
+  describe "#republish" do
+    describe "if the republish command succeeds" do
+      before do
+        publishing_api
+          .given("a unpublished document exists with content_id: #{@content_id}")
```

Comment:
These given's all need to correlate with set-ups in this Publishing API file: https://github.com/alphagov/publishing-api/blob/master/spec/service_consumers/pact_helper.rb

So this one here "a unpublished document exists with content_id: #{@content_id}" doesn't exist whereas we can use: "an unpublished content item exists with content_id: #{@content_id}" that references: https://github.com/alphagov/publishing-api/blob/master/spec/service_consumers/pact_helper.rb#L137

Reply:
Ah okay, i was confuddled by this bit and thought they were just namespaces